data/maps/landmarks.asm

```diff
@@ -103,6 +104,7 @@ Landmarks:
 	landmark  20, 100, Route27Name
 	landmark  12, 100, TohjoFallsName
 	landmark  20,  68, Route28Name
+	assert_table_length OTHER_LANDMARK
```

Comment:
You could add an `assert_table_length KANTO_LANDMARK`, but I don't think they need to be here -- every line is supposed to pair up with its constant already, so these asserts wouldn't catch extra errors.
```suggestion
```